### ISSUE TITLE
About High DPI Support

### ISSUE THREAD

**Issue by edips** [lutraconsulting/input#392]
Hi. I ask the question to make sure a problem about QGIS map canvas. 
I have a qgs viewer app based on Input, I used QQC2 Material. When I use Qt's high DPI, map canvas shows object pixelized and when I do fast zoom-in and zoom-out for big projects, the app crashes. However when I remove high DPI scaling, the app doesn't crash and the map looks normal resolution. I didn't understand why it crashes when I use HDPI in main.cpp.
Custom DPI scaling parameters are used for Input. Why aren't Qt's default high DPI scaling used for Input? Like: 
```QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling)```
Is it because of affecting QGIS map canvas?
Thanks.

**Comment by wonder-sk** [comment]
Hmm I am not sure what's the problem there. Do you have a backtrace of the crash? And does it crash on desktop / on mobile only or both?

**Comment by edips** [comment]
I had java activity error messages on mobile. It only crashes on mobile for my app when zooming very fast. I tested on desktop for big projects like 3D styled city sample, I zoomed in and out very fast, it didn't crash. I will enable HDPI and will test on Input for main.cpp. I will report here. Thanks.

**Comment by edips** [comment]
Input also crashed when enabling HDPI and I get the error message:
```F libc    : Fatal signal 6 (SIGABRT), code -6 (SI_TKILL) in tid 8244 (Thread (pooled)), pid 8139 (lutraconsulting)```

**Comment by edips** [comment]
I built Input on Qt Creator with local Osgeo4A builds on Qt 5.12.4. I think somethink affected QGIS libraries such as QGIS, NDK, SDK or QT version so it crashed. But then I built apk from lutraconsulting/osgeo4a docker image. It didn't crashed even if I enabled HDPI, only resolution of map canvas looks lower. Tested demo is [South Perth](https://github.com/qgis/demo.qgis.org/tree/master/demos/south_perth) :)
